pkgs/development/libraries/libfive/default.nix

```diff
@@ -65,6 +70,18 @@ stdenv.mkDerivation {
     ln -s "$out/bin/Studio" "$out/bin/libfive-studio"
   '';
 
+  postFixup = ''
+    # Create links so Python is happy.
+    mkdir -p "$out/lib/python/libfive/src"
+    ln -s "$out/lib/libfive.so" "$out/lib/python/libfive/src/libfive.so"
+    mkdir -p "$out/lib/python/libfive/stdlib"
+    ln -s "$out/lib/libfive-stdlib.so" "$out/lib/python/libfive/stdlib/libfive-stdlib.so"
+
+    # Create links so studio is happy.
+    mkdir -p "$out/libfive/bind"
+    ln -s "$out/lib/python" "$out/libfive/bind/python"
+  '';
+
```

Comment:
may not needed once converted

Reply:
This is by far the most confusing part of this to me - libfive does some really strange things when sourcing the libraries. I'm not sure if the path constructed [here](https://github.com/libfive/libfive/blob/master/libfive/bind/python/libfive/ffi.py#L49) for the bindings can be easily circumvented. It does appear that there is an option to utilize an environment variable, should that be the preferred approach? Or should this get symlinked to this subdirectory the python bindings are in? The libfive-studio application also does some strange [sourcing](https://github.com/libfive/libfive/blob/master/studio/src/python/interpreter.cpp#L209) when trying to find the bindings, which is why the symlinks for studio exist. Honestly I'd be happy if there was a simple way for Nix to handle this, but I don't understand python packaging well enough to know the correct approach here.

Reply:
This has been changed, but is still needed.